cx_Freeze/freezer.py

```diff
@@ -845,6 +631,429 @@ def Freeze(self):
+        """Called for copying certain top dependencies.  We need this as a separaet function
+        so that it can be overriden on Darwin."""
+
+        # this recovers the cached MachOReference pointers to the files
+        # found by the _GetDependentFiles calls made previously (if any).
+        # If one is found, pass into _CopyFile.
+        # We need to do this so the file knows what file referenced it,
+        # and can therefore calculate the appropriate rpath.
+        # (We only cache one reference.)
+        cachedReference = self.darwinTracker.getCachedReferenceTo(
+            sourcePath=source
+        )
+        self._CopyFileRecursion(
+            source,
+            target,
+            copyDependentFiles=True,
+            includeMode=True,
+            machOReference=cachedReference,
+        )
+        return
+
+    def _GetDefaultBinPathExcludes(self):
+        return ["/lib", "/usr/lib", "/System/Library/Frameworks"]
+
+    def _GetDependentFiles(self, path, darwinFile: Optional[DarwinFile] = None) -> List[str]:
+        path = os.path.normcase(path)
+        dependentFiles = self.dependentFiles.get(path, None)
+        if dependentFiles is not None:
+            return dependentFiles
+
+        # if darwinFile is None (which means that _GetDependentFiles is
+        # being called outside of _CopyFile -- e.g., one of the
+        # preliminary calls in _FreezeExecutable), create a temporary
+        # DarwinFile object for the path, just so we can read its
+        # dependencies
+        if darwinFile is None:
+            darwinFile = DarwinFile(
+                originalFilePath=path, referencingFile=None
+            )
+        dependentFiles = darwinFile.getDependentFilePaths()
+
+        # cache the MachOReferences to the dependencies, so they can be
+        # called up later in _CopyFile if copying a dependency without
+        # an explicit reference provided
+        # (to assist in resolving @rpaths)
+        for reference in darwinFile.getMachOReferenceList():
+            if reference.isResolved():
+                self.darwinTracker.cacheReferenceTo(
+                    sourcePath=reference.resolvedReferencePath,
+                    machOReference=reference,
+                )
+        self.dependentFiles[path] = dependentFiles
+        return dependentFiles
+
+    def _PlatformExecutableDependencyDir(self) -> str:
+        return os.path.join(self.targetdir, "lib")
+
+class LinuxFreezer(Freezer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.patchelf = Patchelf()
+        return
+
+    def _PostCopyHook(self,
+                      source,
+                      target,
+                      normalizedSource,
+                      normalizedTarget,
+                      copyDependentFiles,
+                      includeMode=False):
+        # TODO: remove usages of source/target (only use normalied paths)
```

Comment:
You can remove the TODO. At least I need the raw source here, not the normalized.

Reply:
Done. Removed.